README.md

```diff
@@ -1,24 +1,33 @@
-# Babylon React Native
-
-This project provides Babylon Native integration into React Native.
-
 [![](https://github.com/BabylonJS/BabylonReactNative/workflows/Publish%20Package/badge.svg)](https://github.com/BabylonJS/BabylonReactNative/actions?query=workflow%3A%22Publish+Package%22)
 [![npm version](https://badge.fury.io/js/%40babylonjs%2Freact-native.svg)](https://badge.fury.io/js/%40babylonjs%2Freact-native)
-[![BabylonNative master branch update](https://github.com/BabylonJS/BabylonReactNative/actions/workflows/bn_master_commit.yml/badge.svg)](https://github.com/BabylonJS/BabylonReactNative/actions/workflows/bn_master_commit.yml)
+
+# Babylon React Native
+
+Babylon React Native brings the power and flexibility of [Babylon.js](https://github.com/BabylonJS/Babylon.js) into a [React Native](https://github.com/facebook/react-native) environment via [Babylon Native](https://github.com/BabylonJS/BabylonNative).
 
 ## Current Status
 
-Babylon React Native is in the early phase of its development, and has the following limitations:
+### Limitations
 
-1. Android and iOS support only - support for Windows is planned, but the timeline is currently unknown.
+1. Android and iOS only - support for Windows is experimental.
 1. Touch input only - mouse, keyboard, and controllers are not yet supported.
 1. Single view only - multiple views are not yet supported (only a single view can be displayed).
 
-It is also worth noting that Babylon React Native relies heavily on newer React Native constructs including JSI to get the performance characteristics required for real time rendering. JSI allows for direct synchronous communication between native code and JavaScript code, but is incompatible with "remote debugging." If you need to debug your JavaScript code that uses Babylon React Native, you should enable Hermes and use "direct debugging" (e.g. chrome://inspect or edge://inspect). See the [React Native documentation](https://reactnative.dev/docs/hermes) for more info.
+### What is Supported from Babylon.js
+
+See [this](https://github.com/BabylonJS/BabylonNative#what-is-supported-from-babylonjs) from the Babylon Native repo.
+
+### Performance
+
+React Native applications targeting [JavaScriptCore on iOS](https://reactnative.dev/docs/javascript-environment#javascript-runtime) or [Hermes](https://reactnative.dev/docs/hermes) will not have JIT enabled. This may cause performance issues for JavaScript heavy scenarios.
```

Comment:
Even on Android I don't think there is any way to enable JIT with any JS engine.

Reply:
I don't think so either, but that's not what the React Native docs say, so I'm just repeating it.